capture/genBitmaps.js

```diff
@@ -158,7 +158,7 @@ function processScenario (casper, scenario, scenarioOrVariantLabel, scenarioLabe
         scenario.removeSelectors.forEach(function (o, i, a) {
           casper.evaluate(function (o) {
             Array.prototype.forEach.call(document.querySelectorAll(o), function (s, j) {
-              s.style.display = 'none';
+                s.remove();
```

Comment:
This will only work if jQuery or similar is included in the target page.  I don't think we are injecting that.  Please review.